lib/iris/tests/test_grib_save.py

```diff
@@ -34,7 +34,7 @@
 
 if tests.GRIB_AVAILABLE:
     import gribapi
```

Comment:
ECCodes provides a module called gribapi, it is a replacement for GRIB_API

see
https://software.ecmwf.int/wiki/display/ECC/GRIB-API+migration
for more information

thus, the import name is unchanged